languages/rust/src/fsm/led.rs

```diff
@@ -1,43 +1,14 @@
-#![allow(non_upper_case_globals)]
+use lf;
 
-use ::{
-    Flipper,
-    StandardModuleFFI,
-    ModuleFFI,
-    _lf_module,
-};
+pub struct Led;
```

Comment:
I think we need to find some way to provide both an instance-level API and a module-level (similar to Java static) API such that the module level `Led::rgb` uses the default device and the instance-level `led.rgb` will use a specific device.